mantidproject/mantid
# PDcharacterizations support named containters

This PR adds the ability to have a second characterizations file which connects specific sample containers to characterization runs. 

**To test:**

The files mentioned are the ones in the unit tests. This script allows you to explore the property manager and see what happens with different settings.

```python
name = 'PG3_37005'
LoadEventNexus(Filename=name, OutputWorkspace=name)
charFiles = ','.join(['ExternalData/Testing/Data/UnitTest/Test_characterizations_char.txt',
                      'ExternalData/Testing/Data/UnitTest/PG3_char_2016_02_15-PAC-single.txt'
PDLoadCharacterizations(Filename=charFiles,
                        OutputWorkspace='char')

canname = mtd[name].run()['SampleContainer'].value[0]
print 'can="%s"' % canname

PDDetermineCharacterizations(InputWorkspace=name, Characterizations='char ')

# without pac information this should be 15040
propman = PropertyManagerDataService.retrieve('__pd_reduction_properties' )
for key in propman.keys():
  print key, '=', propman[key].value
```

*There is no associated issue.*

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
